docs/reference/master-commandline-reference.md

```diff
@@ -181,6 +181,22 @@ Example:
 nfd-master -enable-nodefeature-api=false
 ```
 
+### -enable-nodefeaturegroup-api
+
+> **NOTE:** This feature is experimental and may change in the future.
+
+The `-enable-nodefeaturegroup-api` flag enables/disables the
+[NodeFeatureGroup](../usage/custom-resources.md#nodefeaturegroup) CRD API for
+working with NodeFeatureGroups.
+
+Default: false
+
+Example:
+
+```bash
+nfd-master -enable-nodefeaturegroup-api=false
+```
```

Comment:
Since the default value is false, I suggest:
```suggestion
```bash
nfd-master -enable-nodefeaturegroup-api=true
```
```